apex_launchtest/test/test_domain_coordinator.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2019 Apex.AI, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import apex_launchtest.domain_coordinator
+
+
+class TestUniqueness(unittest.TestCase):
+
+    def test_quickly(self):
+        # Quick and simple test to see that we generate unique domains.  Will not necessarily
+        # find problems because domains are selected randomly.  We're only asking for 10
+        # domains out of 100 so most of the time we'll probably get lucky
+        domains = []
+
+        for _ in range(10):
+            domains.append(apex_launchtest.domain_coordinator.get_coordinated_domain_id())
```

Comment:
@pbaughman why not using a list comprehension e.g.
```py
domains = [apex_launchtest.domain_coordinator.get_coordinated_domain_id() for _ in range(10)]
``` 
instead ?

Reply:
I had it in my head that list comprehensions were lazy evaluated, but they're not.